### ISSUE TITLE
hide Web Share button on desktop

### ISSUE THREAD

**Issue by dwelle** [excalidraw/excalidraw#3224]
It has limited usefuless, and in some browsers/OS it acts as supported, but the button doesn't do anything.

**Comment by tomayac** [comment]
Does this happen on the Edge Stable branch without any `edge://flags` set?

**Comment by tomayac** [comment]
There is feature detection happening: `"share" in navigator`.

**Comment by dwelle** [comment]
> Does this happen on the Edge Stable branch without any `edge://flags` set?

Yes. On Win 7, tho, but it still has a significant market share.

In the PR you also said something about `navigator.share` possibly being defined without actual support — not sure what it was referring to.

In either case, on Deskop this API doesn't make that much sense anyway, so if we can't guarantee that displaying the button will also mean it's working, I'm for using it only on Mobile.

**Comment by tomayac** [comment]
Desktop support is coming to Chrome/Edge, here is the Mac-specific bug: https://bugs.chromium.org/p/chromium/issues/detail?id=1144920. On non-stable it might be exposed already, without doing anything yet.

**Comment by tomayac** [comment]
For a currently working desktop integration, try it on macOS Safari.
